python/pyspark/sql/functions.py

```diff
@@ -1973,6 +1973,11 @@ def slice(x, start, length):
     """
     Collection function: returns an array containing  all the elements in `x` from index `start`
     (array indices start at 1, or from the end if `start` is negative) with the specified `length`.
+
+    :param x: the array to be sliced
+    :param start: the starting index
+    :param length: the length of the slice
```

Comment:
Okay, that's fine but can you at least update R and Scala APIs too to be consistent? You can add new params for start and length at https://github.com/apache/spark/blob/master/R/pkg/R/functions.R#L213 and add `@param`s at https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L3267-L3272

Reply:
Ok! will update shortly

Reply:
I added the scala params. 

Maybe I am mistaken, but it seems the R API already has this documented at https://github.com/apache/spark/blob/a9fbd310300e57ed58818d7347f3c3172701c491/R/pkg/R/functions.R#L3623

The rendering seems to work fine too on https://spark.apache.org/docs/latest/api/R/index.html so I am leaving the R docs alone.

Reply:
Thanks. Can you match the wording while we're here?

Reply:
Fixed.